go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/cyberark/conjur-api-go v0.5.2
 	github.com/cyberark/conjur-authn-k8s-client v0.13.0
 	github.com/cyberark/summon v0.7.0
+	github.com/denisenkom/go-mssqldb v0.0.0-00010101000000-000000000000
```

Comment:
This dependency needs LICENSES check/update.

CC: @izgeri

Reply:
I'm glad to see it here, because for prior connectors we haven't had it be this obvious.

it's good to get notified of it here in the PR, but we don't need to make the fix in practice until we're ready for the next tag. I expect that will be within 2 weeks, to include whatever fixes we discover are needed as part of perf testing and the initial MSSQL connector.